stubs/DuskTestCase.stub

```diff
@@ -28,8 +29,19 @@ abstract class DuskTestCase extends BaseTestCase
      */
     protected function driver()
     {
+        $options = (new ChromeOptions)->addArguments([
+            '--headless',
+            '--disable-gpu',
```

Comment:
this is here as there is a recommendation to add this for now here: https://developers.google.com/web/updates/2017/04/headless-chrome